src/main/target/MOTOLAB/target.c

```diff
@@ -23,14 +23,14 @@
 #include "drivers/timer.h"
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    { TIM3,  IO_TAG(PA4),  TIM_Channel_2, 1, IOCFG_AF_PP,    GPIO_AF_2, TIM_USE_MC_MOTOR }, // PWM1  - PA4  - *TIM3_CH2
-    { TIM3,  IO_TAG(PA6),  TIM_Channel_1, 1, IOCFG_AF_PP,    GPIO_AF_2, TIM_USE_MC_MOTOR }, // PWM2  - PA6  - *TIM3_CH1, TIM8_BKIN, TIM1_BKIN, TIM16_CH1
-    { TIM3,  IO_TAG(PB0),  TIM_Channel_3, 1, IOCFG_AF_PP,    GPIO_AF_2, TIM_USE_MC_MOTOR }, // PWM3  - PB0  - *TIM3_CH3, TIM1_CH2N, TIM8_CH2N
-    { TIM3,  IO_TAG(PB1),  TIM_Channel_4, 1, IOCFG_AF_PP,    GPIO_AF_2, TIM_USE_MC_MOTOR }, // PWM4  - PB1  - *TIM3_CH4, TIM1_CH3N, TIM8_CH3N
-    { TIM2,  IO_TAG(PA1),  TIM_Channel_2, 1, IOCFG_AF_PP,    GPIO_AF_1, TIM_USE_MC_MOTOR }, // PWM5  - PA1  - *TIM2_CH2, TIM15_CH1N
-    { TIM2,  IO_TAG(PA2),  TIM_Channel_3, 1, IOCFG_AF_PP,    GPIO_AF_1, TIM_USE_MC_MOTOR }, // PWM6  - PA2  - *TIM2_CH3, !TIM15_CH1
-    { TIM15, IO_TAG(PA3),  TIM_Channel_2, 1, IOCFG_AF_PP,    GPIO_AF_9, TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO }, // PWM7  - PA3  - *TIM15_CH2, TIM2_CH4
-    { TIM1,  IO_TAG(PA8),  TIM_Channel_1, 1, IOCFG_AF_PP,    GPIO_AF_6, TIM_USE_MC_MOTOR | TIM_USE_MC_SERVO }, // PWM8  - PA8  - *TIM1_CH1, TIM4_ETR
+    { TIM3,  IO_TAG(PA4),  TIM_Channel_2, 1, IOCFG_AF_PP,    GPIO_AF_2, TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR }, // PWM1  - PA4  - *TIM3_CH2
+    { TIM3,  IO_TAG(PA6),  TIM_Channel_1, 1, IOCFG_AF_PP,    GPIO_AF_2, TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR }, // PWM2  - PA6  - *TIM3_CH1, TIM8_BKIN, TIM1_BKIN, TIM16_CH1
+    { TIM3,  IO_TAG(PB0),  TIM_Channel_3, 1, IOCFG_AF_PP,    GPIO_AF_2, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO }, // PWM3  - PB0  - *TIM3_CH3, TIM1_CH2N, TIM8_CH2N
```

Comment:
Using `TIM3` for both motor and servo is not recommended - this setup won't support OneShot and faster protocols.

Possible fix: use `TIM8_CH2N` and `TIM8_CN3N` for PB0 and PB1 pins. I'm not sure if timer N-channels will work though.

Reply:
Another solution is to force motor/servo protocol to PWM@50HZ if using fixed-wing mixer - https://github.com/iNavFlight/inav/issues/2499#issuecomment-346973939

Reply:
It is the same problem I encountered with the PIKOBLX target. This FC uses the same pins as PWM outputs. Using the timers with complementary channels did not work for me at the time but it is very possible I didn't do everything right.

I just checked the code and using complementary channels seems to be supported but it looks like the timer configuration output field needs to be ORed with the TIMER_OUTPUT_N_CHANNEL macro. See pwm_config.c:129. Can you confirm ?

I made the modifications to use `TIM8_CH2N` and `TIM8_CH3N` for PB0 and PB1 pins in this branch: https://github.com/shellixyz/inav/tree/MOTOLAB_FW_support_alt_timers

I will test this modification soon on the PIKOBLX FC soon.
@jcaamnes could you try on the MOTOLAB FC ?

Reply:
I tested on PIKOBLX but it doesn't work. I get no signal on outputs 3 and 4.
https://github.com/shellixyz/inav/tree/PIKOBLX_alt_timers